fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -56,6 +61,13 @@ def self.show_details(filter: nil)
         print_output_variables(action, filter)
         print_return_value(action, filter)
 
+        if Fastlane::Actions.is_class_action?(action) && action.category == :deprecated
+          puts "==========================================".deprecated
```

Comment:
Everything else in here uses `puts`... the UI methods add timestamps which don't go with the rest of the actions list stuff